.vscode/settings.json

```diff
@@ -0,0 +1,29 @@
+{
+  "search.exclude": {
+    ".storybook/data": true,
+    "dist/": true,
+    "node_modules/": true,
+    "third-party/": true,
+    "vendor/": true
+  },
+  "files.exclude": {
+    "dist/": true,
+    "node_modules/": true,
+    "third-party/": true,
+    "vendor/": true
+  },
+  "editor.formatOnSave": true,
```

Comment:
I have this set to `false` and I still get the format on save (I believe from `editor.codeActionsOnSave`) – is it needed?

Reply:
Looks like you're right—removed! 👍🏻